packages/atomic/cypress/scripts/get_hosted_search_page.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
Any possibility to use just a node script for this ?

If we add a python dependency, it needs to be for a good reason, ie: it's actually impossible to perform the needed task with a technology already used in the repo.

The train of thought is simply to make sure that if a newcomer arrives in the team, and we ask them to setup to work on this project, it's as straightforward as it can be.

Reply:
That make sense. I re-wrote in js.